docs.css

```diff
@@ -0,0 +1,47 @@
+
+body{
+    display: grid;
+    grid-template-columns: 1fr 3fr 1fr;
+    grid-template-areas: 'header header header'
+    'left_menu conten conten';
+}
+.navbar{
+    grid-area: header;
+}
+
+.left_menu{
+    margin: 1rem;
+    grid-area: left_menu;
+    font-size: 1.5rem;
+}
+
+
+a{
+    text-decoration: none;
+    color:var(--primary);
+}
+
+a:hover{
+    color: var(--btn-bg);
+}
+
+.content{
+    margin: 1rem;
+    font-size: 1.5rem;
+    line-height: 2.4rem;
+    align-self: flex-start;
+    grid-area: conten;
+}
+
+/* code snippet */
+.code-snippet{
+    margin: 10px;
+    padding: 10px;
+    border: 1px solid var(--bg-color);
+    border-radius: 10px;
+    background-color: var(--ternary);
```

Comment:
This is great to use css variable

Reply:
@arjun-computer-geek rightly pointed out